.coderabbit.yaml

```
@@ -0,0 +1,70 @@
+  high_level_summary: true
+  high_level_summary_placeholder: '@coderabbitai summary'
+  auto_title_placeholder: '@coderabbitai'
+  changed_files_summary: true
+  sequence_diagrams: true
+  assess_linked_issues: true
+  related_issues: true
+  related_prs: true
+  suggested_labels: true
+  suggested_reviewers: true
+  poem: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)